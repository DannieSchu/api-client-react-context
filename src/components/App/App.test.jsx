import React from 'react';
import { shallow } from 'enzyme';
import App from './App';
import { APIClientProvider } from '../../hooks/APIClientProvider';

describe('App component', () => {
  it('renders App', () => {
    const wrapper = shallow(
      <APIClientProvider>
        <App />
      </APIClientProvider>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
