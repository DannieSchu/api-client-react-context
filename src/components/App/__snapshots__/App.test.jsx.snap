// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<ContextProvider
  value={
    Object {
      "body": "",
      "handleChange": [Function],
      "handleClick": [Function],
      "handleDelete": [Function],
      "handleSubmit": [Function],
      "loading": false,
      "method": "get",
      "requests": Array [],
      "results": null,
      "url": "",
    }
  }
>
  <App />
</ContextProvider>
`;
